### ISSUE TITLE
[Improvement] Different style for when Parinfer is on

### ISSUE THREAD

**Issue by laconiccrafts** [oakmac/sublime-text-parinfer#36]
On Atom, when Parinfer is on, the closing parens, vectors, curly braces, are styled with a light gray color. 

This helps one to know when Parinfer is on or off.

On Sublime Text I offen find myself having to double check it.

**Comment by oakmac** [comment]
Yeah - this would be nice to have. I'm not sure if the Sublime API has support for that? But it's worth looking into.

**Comment by laconiccrafts** [comment]
To be honest, I've never learned their API, for it seems unnecessarily verbose/complex, so I never made the effort to learn it.

I'm also only using Sublime Text for lack of a better editor for Clojure.

But if there's some way to help I'm willing to do it.

:)
